wordpress-mobile/WordPress-iOS
# Implement copy paste of image in GB mobile

Fixes https://github.com/wordpress-mobile/gutenberg-mobile/issues/730

Relate GB-mobile PR: https://github.com/wordpress-mobile/gutenberg-mobile/pull/810

To test:
 - Create a new post using GB-mobile
 - Open the Photos app
- Select one or more images
- Tap on the copy button
- Go back to the post
- Select a text block, and tap paste
- See that the images are inserted in the post has image blocks
- Check that the upload happens correctly.

Update release notes:
